### ISSUE TITLE
Android - There is no focus in compose box after tapping on deep link 

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#16023]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing PR** https://github.com/Expensify/App/pull/15994

## Action Performed:
1. Go to NewDot app
2. Log in with any account
3. Send yourself a link to a public report you're not already a member of on NewDot (from a different account).
4. Click on that link, and verify you can access the report.

## Expected Result:
There is focus in compose box after tapping on deep link 

## Actual Result:
There is no focus in compose box after tapping on deep link 

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.85.1

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/225484849-ede42617-cda0-451a-bb7c-c8b8a2a38254.mp4

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @twisterdotcom (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by dylanexpensify** [comment]
reviewing

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~01d0ad62d564fa664e

**Comment by MelvinBot** [comment]
Current assignee @twisterdotcom is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @Santhosh-Sellavel (`External`)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @Julesssss (`External`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by Julesssss** [comment]
Awaiting proposals

**Comment by tienifr** [comment]
@Julesssss @dylanexpensify I think this is a feature request. Currently on native `There is focus in compose box` only happens if we access a new chat (without any messages). For existing chat it will not focus on the compose box when accessing the chat.

**Comment by Julesssss** [comment]
Ah right. It would be good to confirm that this is only occurring on Andorid and not iOS then.

**Comment by burczu** [comment]
Hey, it's Bartek from Callstack, I would like to take care of this issue

**Comment by MelvinBot** [comment]
📣 @burczu You have been assigned to this job by @mountiny!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by mountiny** [comment]
Awaiting your proposal @burczu thanks! Anyone else also can submit their proposals before Bartek's

**Comment by MelvinBot** [comment]
📣 @BeeMargarida You have been assigned to this job by @mountiny!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by twisterdotcom** [comment]
Just waiting on the PR.

**Comment by BeeMargarida** [comment]
Was the proposal above accepted?

**Comment by Santhosh-Sellavel** [comment]
@Julesssss 
Sorry for the delay here,

I think we should put this on hold and do nothing about this for now. 

#### Reasons

Well, this issue is not specific to deep link flow. It happens in normal cases too. Issue outline focus is inconsistent between mobile native & mweb apps for which we have issue #10731 to unify the focus across platforms and make it consistent on mobile i.e 

> Always focus the input, but prevent keyboard from opening on mobile & mWeb

And we have done some work there on the respective upstream repos and waiting for the release on the RNW.

**Comment by situchan** [comment]
I think this should be added to the Focus Tracking issue https://github.com/Expensify/App/issues/15992

**Comment by Julesssss** [comment]
> @Julesssss that makes sense to me. Should we close and link this to #15992

Yeah, makes sense to me 👍
